scripts/create_changelog.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env python
```

Comment:
Perhaps rename file to: update_changelog_and_credits.py

Reply:
Done.